ImageMagick/Source/Dockerfile

```diff
@@ -0,0 +1,101 @@
+#
+# Creates an image suitable for building the Magick.NET.CrossPlatform project for Linux.
+# 
+# This image requires that ImageMagick and it's dependencies already be checked out to your
+# machine under ImageMagick/Source, ideally using Checkout.cmd.
+
+FROM ubuntu:16.04
+
+# Install packages
+RUN apt-get update
+RUN apt-get install -y gcc g++ make autoconf autopoint pkg-config libtool nasm git openssh-server dos2unix
+
+# Initialize the sshd server
+RUN mkdir /var/run/sshd; chmod 755 /var/run/sshd
+RUN sed -i -e 's/#PasswordAuthentication yes/PasswordAuthentication no/g' /etc/ssh/sshd_config
+COPY authorized_keys /root/.ssh/authorized_keys
+RUN chmod 600 ~/.ssh/authorized_keys
+
+# Build libbz2
+COPY /ImageMagick/bzlib /bzlib
+WORKDIR /bzlib
+RUN make CFLAGS="-Wall -Winline -O3 -fPIC -g -D_FILE_OFFSET_BITS=64"; \
+    make install;
+
+# Build zlib
+COPY /ImageMagick/zlib /zlib
+WORKDIR /zlib
+RUN export CFLAGS="-O3 -fPIC"; \
+    dos2unix configure; \
+    ./configure --static; \
+    make; \
+    make install
+
+# Build lzma
+RUN git clone -n https://git.tukaani.org/xz.git /lzma; 
```

Comment:
Do we really need lzma support? I think I rather have this block removed.

Reply:
I was under the impression it was needed for `TIFF/LZW` support, but I may have been mistaken. Either way that's easy to back out and I can worry about it later if it really does turn out to be necessary for us.